website/app/styles/doc-components/badge.scss

```diff
@@ -12,14 +12,25 @@
   display: inline-flex;
   align-items: center;
   max-width: 100%;
+  line-height: 1;
+  border-radius: 100px;
+}
+
+// sizes
+
+.doc-badge--size-large { // default
   height: 26px;
   padding: 3px 12px 4px 12px;
   font-weight: 600;
   font-size: 14px;
-  line-height: 1;
-  border-radius: 100px;
 }
 
+.doc-badge--size-medium {
+  height: 24px;
```

Comment:
(question, non-blocking) 

1. Why do you use `height` here instead of `min-height` or something else? 
2. Why not use `vh` units here? Any particular reason?

Reply:
> Why do you use height here instead of min-height or something else?

Good point. I've updated the code.

> Why not use vh units here? Any particular reason?

How would that work? when the window resize, the badge would resize too (and collapse) 🤔

Reply:
I am resolving this now.